### PR TITLE
Added template files used by the Optum OSPO

### PR DESCRIPTION
**Pull request by robertcrockett**
Added several files typically used by the Optum OSPO for community contribution and compliance.
- BUG_REPORT.md
- FEATURE_REQUEST.md
- PULL_REQUEST_TEMPLATE.md
- CODE_OF_CONDUCT.md
- CONTRIBUTING.md
- INDIVIDUAL_CONTRIBUTOR_LICENSE.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+We track our work using Issues in GitHub. Feel free to open up your own issue to point out areas for improvement or to suggest your own new experiment. If you are comfortable with signing the waiver linked above and contributing code or documentation, grab your own issue and start working.
+
+## Coding Standards
+
+We have some general guidelines towards contributing to this project.
+
+### Languages
+
+*Lua*
+
```
Comment (htmldoug) [inline CONTRIBUTING.md]:
```suggestion
```

There's no lua here. Looks like copy-paste. Even though it's currently all scala, I wouldn't go so far as to restrict code to scala-only if there's some other jvm code of value, e.g. https://github.com/rallyhealth/weePickle/issues/117#issuecomment-1350169393. WDYT about deleting this section?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* Please read this guide and make sure you agree with our [Contributor License Agreement ("CLA")](INDIVIDUAL_CONTRIBUTOR_LICENSE.md).
+* Make sure git knows your name and email address:
+   ```
+   $ git config user.name "J. Random User"
+   $ git config user.email "j.random.user@example.com"
+   ```
+>The name and email address must be valid as we cannot accept anonymous contributions.
+* Write good commit messages.
+> Concise commit messages that describe your changes help us better understand your contributions.
+* The first time you open a pull request in this repository, you will see a comment on your PR with a link that will allow you to sign our Contributor License Agreement (CLA) if necessary.
```
Comment (htmldoug) [inline CONTRIBUTING.md:34]: Is there a github workflow for this?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+> The link will take you to a page that allows you to view our CLA.  You will need to click the `Sign in with GitHub to agree button` and authorize the cla-assistant application to access the email addresses associated with your GitHub account.  Agreeing to the CLA is also considered to be an attestation that you either wrote or have the rights to contribute the code.  All committers to the PR branch will be required to sign the CLA, but you will only need to sign once.  This CLA applies to all repositories in the Optum org.
+
+## General Guidelines
+
+Ensure your pull request (PR) adheres to the following guidelines:
+
+* Try to make the name concise and descriptive.
+* Give a good description of the change being made. Since this is very subjective, see the [Updating Your Pull Request (PR)](#updating-your-pull-request-pr) section below for further details.
+* Every pull request should be associated with one or more issues. If no issue exists yet, please create your own.
```
Comment (htmldoug) [inline CONTRIBUTING.md]: As a general guideline, I'd like to minimize friction for contributions as much as possible.

What's the motivation for this one? From a github perspective, there's very little difference between issues and pull requests. For example, note that `https://github.com/rallyhealth/weePickle/issues/120` directs to this PR. Was there a time when a PR without a separate issue created a problem?

File: CONTRIBUTING.md (hunk 1 above)
Comment (robertcrockett) [reply CONTRIBUTING.md]: I don't see any issues with that. It is more of an informative section and that is already handled at the repo level view in GitHub.

File: CONTRIBUTING.md (hunk 2 above)
Comment (robertcrockett) [reply CONTRIBUTING.md:34]: There will be once the CLA assistant is turned on at the organization level. Part of this commit has the INDIVIDUAL_CONTRIBUTOR_LICENSE.md file which the CLA assistant will use as a digital contract. That should only show up the first time someone is contributing to the repo under a username.

File: CONTRIBUTING.md (hunk 3 above)
Comment (robertcrockett) [reply CONTRIBUTING.md]: TBH, I don't have the history on that last line. These template files have been around for several years. I think line 47 could be removed.

**Pull request merged**